Add "Tasks" section to Provider App Manual

kevinwadeshaw opened the pull request.

kevinwadeshaw committed:
Add "Tasks" to index.md TOC

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw committed:
Remove "Tasks" from coming soon…

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw committed:
Add new "Tasks" page to Provider App documentation

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw committed:
Copy edits

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw committed:
Copy edits

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw committed:
Reorder sections

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw commented:
@reshmakh this is ready for your review — will add a separate PR for Messages next week 🙃

reshmakh commented:
Can I also ask that we re-record the video once these are addressed? https://youtu.be/xQH27B8sP9o

kevinwadeshaw committed:
Update tasks.md

Updated "Add Task Details" section to include descriptions and examples for Task details options

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw committed:
Update tasks.md with screenshots

Signed-off-by: kevinwadeshaw <33669002+kevinwadeshaw@users.noreply.github.com>

kevinwadeshaw committed:
Update images links and spacing

kevinwadeshaw committed:
Spacing + formatting updates
